### PR TITLE
Added public facing logic for GCP Storage bucket

### PR DESCRIPTION
**Pull request by ganeshjagdale9545**

### CODE REVIEW

File: cartography/intel/gcp/storage.py [hunk 1]
```diff
@@ -32,15 +32,13 @@ def get_gcp_buckets(storage: Resource, project_id: str, common_job_parameters) -
     :return: Storage response object
     """
     try:
+        buckets_list = []
         req = storage.buckets().list(project=project_id)
-        res = req.execute()
-        for item in res.get('items', []):
-            acl = item.get('acl', [])
```
Comment (mpurusottamc) [inline cartography/intel/gcp/storage.py:38]: do we not use `acl` and `defaultObjectAcl` information anywhere?

**Review by mpurusottamc — COMMENTED** [review]
overall code looks good. need one clarification. see comments.

File: cartography/intel/gcp/storage.py (hunk 1 above)
Comment (ganeshjagdale9545) [reply cartography/intel/gcp/storage.py:38]: it is list[dict] object and by this logic overwrite all values and takes only last dict value from list. That’s why it is not useful

Comment (mpurusottamc) [reply cartography/intel/gcp/storage.py:38]: ah! makes sense. merging for deployment.

**Review by mpurusottamc — APPROVED** [review]
looks good. merging for deployment. Let's add this in Inventory project to return public facing attribute, if it already does not exist.

**Pull request merged**